openshift/api.app.yaml

```diff
@@ -0,0 +1,119 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: api
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      service: api
+    name: api
+  spec:
+    replicas: 1
+    selector:
+      service: api
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          service: api
+      spec:
+        containers:
+        - command:
+          - /usr/local/alm/bin/alm
+          env:
+          - name: ALMIGHTY_POSTGRES_HOST
+            valueFrom:
+              secretKeyRef:
+                name: api
+                key: db.host
+          - name: ALMIGHTY_POSTGRES_PORT
+            valueFrom:
+              secretKeyRef:
+                name: api
+                key: db.port
+          - name: ALMIGHTY_POSTGRES_USER
+            valueFrom:
+              secretKeyRef:
+                name: api
+                key: db.user
+          - name: ALMIGHTY_POSTGRES_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: api
+                key: db.password
+          - name: ALMIGHTY_KEYCLOAK_REALM
```

Comment:
If you look at the ENV variables, there much more in `prod-preview` than here. I believe we want this template to be replica of what we have in `prod-preview`.